waterloop-site/src/App.tsx

```diff
@@ -164,7 +164,7 @@ class App extends React.Component<{}, State> {
           <Switch>
             {pageMappings.map((item) => {
               return (
-                <Route exact={item.exact} path={item.path}>
+                <Route key={item.path} exact={item.exact} path={item.path}>
```

Comment:
This is not relevant to this PR, but I just wanted to make this change because the warning in the console was annoying me.